communication.rst: improve wording (#2765)
ansible/ansible-documentation: docs/docsite/rst/community/communication.rst

.. _communication:

****************************************
Communicating with the Ansible community
****************************************

.. contents::
   :local:

Code of Conduct
===============

All interactions within the Ansible Community are governed by our :ref:`code_of_conduct`. Please read and understand it before participating.

.. _ansible_forum:

Forum
=====

> The `Ansible Forum <https://forum.ansible.com>`_ is the **default and recommended** starting point for most community interactions. It's ideal for:

Reviewer: Emphasis is good

* Asking questions and seeking help.
* Participating in development discussions.
* Learning about events and news.

To get started:

* `Register <https://forum.ansible.com/signup?>`_ to join the community.

> * Search by categories and tags to find topics, or start a new one.

Reviewer: ```suggestion
* Search by `categories <https://forum.ansible.com/categories>`_ and `tags <https://forum.ansible.com/tags>`_ to find topics, or start a new one.
```

> * Subscribe only to topics you're interested in.

Reviewer: How?

Author: @gundalow , redesigned it entirely, PTAL

Reviewer: Good use of bold.

Explore `forum groups <https://forum.ansible.com/g>`_ that match your interests. Joining a group often automatically subscribes you to related posts.

.. _bullhorn:

The Bullhorn
------------

> **The Bullhorn** is our community newsletter, published directly in the Forum:

Reviewer: ```suggestion
**The Bullhorn** is our community's weekly newsletter, published directly in the Forum:
```

* **Subscribe:** Click the ``bell`` button under the `Bullhorn category <https://forum.ansible.com/c/news/bullhorn/17>`_ description, then select ``Watching``.
* **Submit News:** See the `About the Newsletter category post <https://forum.ansible.com/t/about-the-newsletter-category/166>`_ for submission guidelines.
* **Questions about the newsletter:** Ask us in the `Ansible Social room on Matrix <https://matrix.to/#/#social:ansible.com>`_.

Regional and Language-specific channels
---------------------------------------

> Communicate in your preferred language by visiting the `International Communities forum category <https://forum.ansible.com/c/other-languages/10>`_. Current subcategories include Español (Spanish), Português (Portuguese), and Norsk (Norwegian).

Reviewer: Bullet point list stands out more when people skim the page.

```suggestion
Communicate in your preferred language by visiting the `International Communities forum category <https://forum.ansible.com/c/other-languages/10>`_. Current subcategories include:
* Español (Spanish)
* Português (Portuguese)
* Norsk (Norwegian)
```

Author: Good one, thanks
